packages/react/src/components/Embed/Embed.tsx

```diff
@@ -124,7 +124,7 @@ class Embed extends AutoControlledComponent<WithAsProp<EmbedProps>, EmbedState>
 
     if (iframeNil || (!iframeNil && !this.state.active)) {
       this.setState({ active: !this.state.active })
-      _.invoke(this.props, 'onActiveChanged', e, { ...this.props, active: !this.state.active })
+      _.invoke(this.props, 'onActiveChange', e, { ...this.props, active: !this.state.active })
```

Comment:
Can you save the value that we want to add in the state in a variable and reuse it in both setState and here? This is not save, as setState is async, so we should either move this into the callback in setState, or add here a variable's value, not the state's value